pkg/controller/external_secrets/constants.go

```diff
@@ -34,9 +34,17 @@ const (
 	externalsecretsImageVersionEnvVarName = "OPERAND_EXTERNAL_SECRETS_IMAGE_VERSION"
 
 	// externalsecretsImageEnvVarName is the environment variable key name
-	// containing the image version of the external-secrets as value.
+	// containing the image name of the external-secrets as value.
 	externalsecretsImageEnvVarName = "RELATED_IMAGE_EXTERNAL_SECRETS"
 
+	// bitwardenImageEnvVarName is the environment variable key name
+	// containing the image name of the bitwarden-sdk-server as value.
+	bitwardenImageEnvVarName = "RELATED_IMAGE_BITWARDEN_SDK_SERVER"
```

Comment:
looks like we're taking additional maintenance to have to build  bitwarden-sdk-server downstream here with this/

Reply:
yeah, since it's another feature supported by the solution, we had to.